src/main/java/com/epam/test_generator/entities/results/ResultTrait.java

```diff
@@ -12,7 +12,7 @@
      * @param results collection of {@link Status}
      * @return results evaluated by given rules
      */
-    static Status computeStatus(List<? extends AbstractResult> results) {
+    static Status calculateStatus(List<? extends AbstractResult> results) {
```

Comment:
Can we make it not `static`? By declaring this method as `static` we are limiting ourselves in several ways:

- We can't mock it in tests
- We can't use injected dependencies

Reply:
Of course I can make this method as a default, but I'm not really sure about it. I did it static because i thought this behavior is inner and methods shouldn't show outside in implementations.

Reply:
changed to default